packages/camera/camera_avfoundation/ios/camera_avfoundation/Sources/camera_avfoundation/CameraPlugin.swift

```diff
@@ -169,6 +159,35 @@ extension CameraPlugin: FCPCameraApi {
     }
   }
 
+  private func platformLensDirection(for device: FLTCaptureDevice) -> FCPPlatformCameraLensDirection
+  {
+    switch device.position {
+    case .back:
+      return FCPPlatformCameraLensDirection.back
```

Comment:
nit: can just infer the type with `.back`

Reply:
Switched to inferred types for both helpers.